nan.h

```diff
@@ -58,10 +58,14 @@
 # pragma warning( disable : 4530 )
 # include <string>
 # include <vector>
+# include <queue>
+# include <utility>
```

Comment:
Do any of these cause warning 4530, or could they be moved to the unconditional include section?

Reply:
I moved `<utility>` to the unconditional include section, but `<queue>` actually does generate warning 4530